### DISCUSSION TITLE
Quick question about licenses

### DISCUSSION THREAD

**Discussion by cptvideo** [airsdk/Adobe-Runtime-Support#1361] [General]
Just before purchasing a license --  how is the license file employed if I'm building with Animate? 

Also, is the license delivered in the U.S. immediately or is there a wait?

thanks. c.u.

**Comment by ajwfrost** [comment]
Hi

The license file works the same whether you're using Animate or any other IDE, or the AIR SDK command-line tools. Animate would need to use an AIR SDK to build/package up AIR applications so it will call the SDK build tools and - if you're using AIR 33+ - these will look to see if there's a license file present.

For the US, we need to check on the tax requirements with our US finance team so there can be a delay of a day or two in order to get the appropriate tax rate and send over an invoice, which can then be paid by PayPal.. but currently we're not able to offer direct PayPal payments from the website which are available from other locations.

thanks
